README.md

```diff
@@ -1467,3 +1468,13 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents.
+
+- [Azul Overview](#1-architecture-overview)  
+- [Contributing Guidlines](CONTRIBUTING.rst)
+- [Getting Started](#2-getting-started)
+- [Gitlab CI/CD Instance](https://gitlab.dev.singlecell.gi.ucsc.edu/)
+- [Allspark CI/CD Instance](https://allspark.dev.data.humancellatlas.org/)
```

Comment:
These aren't documents. But you do need to request access to them.

Reply:
Yeah, i moved the links to the CI/CD section, then reference them.